SDL2-CS.dll.config

```
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
-	<dllmap dll="SDL2.dll" os="!windows" target="libSDL2-2.0.so.0" />
-	<dllmap dll="SDL2.dll" os="openbsd" target="libSDL2.so" />
-	<dllmap dll="SDL2.dll" os="osx" target="libSDL2.dylib"/>
+	<dllmap dll="SDL2" os="!windows" target="SDL2.so" />
```

This is wrong.
```suggestion
	<dllmap dll="SDL2" os="!windows" target="libSDL2-2.0.so.0" />
```

or

```suggestion
	<dllmap dll="SDL2" os="!windows" target="libSDL2.so" />
```

are the correct so names.